### PR TITLE
Improve Dependencies

### PR DESCRIPTION
**Pull request by dr5hn**
# This PR improves
* Update dependencies to latest
* Add an action to update dependency every monday and raise a pull request

### CODE REVIEW

**Comment by dr5hn** [comment]
This PR is failing build due to issues raised in #17 , Once #17 is merged, we need to rebase this branch and this branch will build perfectly. 👍🏼

File: .github/workflows/update.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+name: Update NPM Dependencies
+
```
Comment (Tomburgs) [inline .github/workflows/update.yml:2]: What's the difference between using this & [dependabot.yml](https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/configuration-options-for-dependency-updates)?

Could do something like this with dependabot
```
updates:
  - package-ecosystem: "npm"
    directory: "/"
    schedule:
      interval: "daily"
```

**Review by Tomburgs — COMMENTED** [review]
Thank you for this PR!

I'm against updating all dependencies right now, I think this should be thoroughly tested to ensure nothing gets broken. Are you sure this is necessary?

In my opinion adding just the dependency update workflow would be sufficient to ensure that our dependencies are up-to-date.

File: .github/workflows/update.yml (hunk 1 above)
Comment (dr5hn) [reply .github/workflows/update.yml:2]: @Tomburgs According to my knowledge, the only difference is
* the dependabot scans package-lock.json/yarn.lock and updates the dependencies, which also includes child dependencies. Updating child dependency of any package may break the package itself.
* this action only scans package.json and updates the dependencies.

**Comment by dr5hn** [comment]
> Thank you for this PR!
> 
> I'm against updating all dependencies right now, I think this should be thoroughly tested to ensure nothing gets broken. Are you sure this is necessary?
> 
> In my opinion adding just the dependency update workflow would be sufficient to ensure that our dependencies are up-to-date.

I've thoroughly tested the website after updating these dependencies and it works perfect. 👍🏼 

I am also okay if you don't want to update them.

**Comment by dr5hn** [comment]
@Tomburgs Let me know what you want to keep and what you want to discard ?

**Comment by Tomburgs** [comment]
@dr5hn I think dependabot would do everything we need, it was created for this very reason.
I've created an issue #20, I'll assign this issue to you, but if you don't want to work on it please let me know.

**Pull request closed without merging**